docs/GettingStarted.md

```diff
@@ -150,23 +150,17 @@ Here's a code sample showing how the default values for built-in devices can be
 	    String framesPerBurstStr = myAudioMgr.getProperty(AudioManager.PROPERTY_OUTPUT_FRAMES_PER_BUFFER);
 	    int defaultFramesPerBurst = Integer.parseInt(framesPerBurstStr);
 
-	    native_setDefaultSampleRate(defaultSampleRate);
-	    native_setDefaultFramesPerBurst(defaultFramesPerBurst);
+	    native_setDefaultStreamValues(defaultSampleRate, defaultFramesPerBurst);
 	}
 
 *jni-bridge.cpp*
 
 	JNIEXPORT void JNICALL
-	Java_com_google_sample_oboe_hellooboe_MainActivity_native_1setDefaultSampleRate(JNIEnv *env,
+	Java_com_google_sample_oboe_hellooboe_MainActivity_native_1setDefaultStreamValues(JNIEnv *env,
```

Comment:
Why is there a '1' in  "native_1set"?

Reply:
It's a weird quirk of defining method names in JNI. If your method name in Java is `native_something` then your JNI declaration must have a "1" after the "_" so `native_1something`. Prefixing the method names with `native_` is good practice because you can easily identify native methods in your Java code.